internal/tutils/psql.go

```diff
@@ -42,7 +42,6 @@ func NewPSQLContainer() (*PSQLContainer, error) {
 	}
 
 	name := fmt.Sprintf("image_builder_test_%d", time.Now().Unix())
-	rand.Seed(time.Now().Unix())
```

Comment:
This is deprecated in 1.20 and linter was complaining, seeding is automatic in 1.20+. The container is already on ubi9 1.20 so this is safe.